### DISCUSSION TITLE
I can't join Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by Sazid99246** [community/community#50767] [GitHub Education]
Hello everyone. I want to join the GitHub student developer pack. I have already submitted my applications more than five times but got rejected. 
My university (Green University of Bangladesh) provides student mail, but the domain doesn't have .edu. After that, I also tried with my ID Card several times. It includes all the things they wanted. But when I tried to submit it,  they told me something like the name of my GitHub profile and the name on the id Card doesn't match, but it matches. Another reason they told me is that the institution on the card doesn't match with what I have provided them. What should I do now to join it?

**Comment by samus-aran** [comment]
Hi @Sazid99246 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Hope this helps! Let us know how you get along 👍
